### PR TITLE
Delete TODOs from the documentation

### LINKED ISSUE #33: Delete TODOs spread across the documentation
Delete TODOs spread across the [documentation](https://ontouml.readthedocs.io/en/latest/todo.html). These should be turned into GitHub issues instead allowing them to be assigned and tracked.

### PR DESCRIPTION
**Pull request by MarekSuchanek**
closes #33 

Created issues: #38, #39, #40, #41

### CODE REVIEW

File: conf.py [hunk 1]
```diff
@@ -39,11 +39,8 @@
 # extensions coming with Sphinx (named 'sphinx.ext.*') or your custom
 # ones.
 extensions = [
-    'sphinx.ext.intersphinx',
```
Comment (claudenirmf) [inline conf.py:42]: Was `sphinx.ext.intersphinx` this being used before?

Comment (MarekSuchanek) [reply conf.py:42]: No, probably was there as a default one... This extension can be used to interlink with other Sphinx documentations, which is not useful for us at the moment.

File: conf.py [hunk 2]
```diff
@@ -39,11 +39,8 @@
...
-    'sphinx.ext.todo',
     'sphinx.ext.mathjax',
     'sphinx.ext.ifconfig',
-    'sphinx.ext.doctest',
```
[the remark this answers is not in the document]
Comment (MarekSuchanek) [reply conf.py:46]: No, again probably was there as a default one... This extension can be used to test if Python code examples are working across the documentation - we don't need that.

**Comment by MarekSuchanek** [comment]
> Let's delete the files `dependency.rst` and `objects_events.rst` for now and change the issues #40 and #41 to be about adding these sections to the documentation. Not only should not have empty sections, but the structure for explaining this concepts may be different from this.
> 
> If you know of other empty sections, let's delete them as well, but I went through the project and couldn't thing any.

Empty pages deleted and issues are rephrased... there are no more empty pages.

**Review by claudenirmf — APPROVED** [review]

**Comment by claudenirmf** [comment]
Btw, if you amended your original commit when deleting the other files, there is no need for that. Feel free to make as many commits as you need on the issue branch because we do a "Squash and Merge" on the PR.

**Comment by claudenirmf** [comment]
Is there a place for us to check the status of the deployment of the documentation?

**Pull request merged**